microovn/database/schema.go

```diff
@@ -36,3 +37,27 @@ CREATE TABLE services (
 
 	return err
 }
+
```

Comment:
Sorry for tagging you directly @masnax but I'm having an issue with this schema migration (independently of any other change in this PR) and you were very knowledgeable about the `microcluster` last time we spoke, so  I was hoping you could maybe point me in a right direction.
About 1/10th of a bootstrap attempts fail with
```
Error: failed to apply update 1: no such table: services
```
I'd hate to jump to conclusions, but I don't see how this error could originate just from the statements in the new transaction. Is it possible that there's some issue with the way `microcluster` executes schema updates so that the new update (below) is executed before the first update (above) properly finishes?
(@fnordahl just 'cause we talked about this yesterday)

Reply:
You are the best! Thank you very much